### PR TITLE
OLM tests using CLI

### PR DESCRIPTION
**Pull request by famartinrh**
### Type of change

- Refactoring

### Description

This PR refactors the OLM tests and uses the CLI to install the operator instead of using the web UI, this should guarrantee better stability in the long term

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [x] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by kornys** [comment]
@famartinrh just question, does fabric8 client 4.6.2 contain builders for subscriptions and operatorsource? It would be nice to use it if it exists instead of using json and executor :)

**Comment by famartinrh** [comment]
@kornys I also thought about that and the answer is no, I checked that and those objects are not implemented in fabric8

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 1]
```diff
@@ -34,9 +49,50 @@
 @OpenShift(version = 4)
 class OperatorLifecycleManagerTest extends TestBase implements ITestIsolatedStandard {
     private static Logger log = CustomLogger.getLogger();
-    private final String marketplaceNamespace = "openshift-marketplace";
     private final String infraNamespace = "openshift-operators";
 
+    private static final int CREATE_CR_TIMEOUT_MILLIS = 30000;
+
+    //amqonline.1.3.1
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: more refs

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 2]
```diff
@@ -34,9 +49,50 @@
...
+    private static String csvName;
+
+    //amq-online-operator
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: remove

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 3]
```diff
@@ -34,9 +49,50 @@
...
+    private static String operatorSource;
+    //amq-online
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: remove

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 4]
```diff
@@ -34,9 +49,50 @@
...
+    private static String operatorName;
+
+    private static String subscriptionName = "systemtests-enmasse-operator";
+
+    @BeforeAll
+    void prepare() throws Exception {
+        String productName = Environment.getInstance().isDownstream() ? "amq-online" : "enmasse";
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: Take this from the APP_NAME env (see Environment.java)

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 5]
```diff
@@ -92,16 +198,15 @@ void testBasicMessagingAfterOlmInstallation() throws Exception {
...
 
     @Test
-    @SeleniumFirefox
     @Order(4)
     void uninstallOperator() throws Exception {
         TestUtils.cleanAllEnmasseResourcesFromNamespace(infraNamespace);
-        Openshift4WebPage page = new Openshift4WebPage(SeleniumProvider.getInstance(), getOCConsoleRoute(), clusterUser);
-        page.openOpenshiftPage();
-        page.uninstallFromCatalog(environment.getAppName());
+        //oc delete subscriptions -n openshift-operators amq-online-operator
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: Remove reference

File: systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java [hunk 6]
```diff
@@ -92,16 +198,15 @@ void testBasicMessagingAfterOlmInstallation() throws Exception {
...
+        KubeCMDClient.runOnCluster("delete", "subscriptions", "-n", infraNamespace, subscriptionName);
+        //oc delete csv -n openshift-operators amqonline.1.3.0
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/olm/OperatorLifecycleManagerTest.java]: Remove reference

**Pull request merged**